crates/astria-sequencer/src/api_state_ext.rs

```diff
@@ -381,17 +381,10 @@ mod test {
 
     use std::collections::HashMap;
 
-    use astria_core::{
-        sequencer::v1::{
-            asset::Id,
-            block::{
-                Deposit,
-                SequencerBlock,
-            },
-            Address,
-            RollupId,
-        },
-        Protobuf,
+    use astria_core::sequencer::v1::{
+        asset::Id,
+        block::Deposit,
+        Address,
```

Comment:
Why do we have changes here, when no change in the rest of file?

Reply:
compiler was giving me unused import warnings so i just removed them